Resources/Prototypes/Entities/Objects/Specific/Hydroponics/tools.yml

```diff
@@ -18,6 +18,8 @@
         Slash: 10
   - type: Item
     sprite: Objects/Tools/Hydroponics/hoe.rsi
+  - type: StaticPrice
+    price: 35
```

Comment:
Pretty sure all the hydro tools are lathe manufacturable so should be worth almost nothing.

Reply:
done